node/Cargo.toml

```diff
@@ -23,6 +23,7 @@ bitcoincore-rpc = { version = "0.19.0"}
 bitcoincore-rpc-json = { version = "0.19.0"}
 bitcoincore-zmq = { version = "1.5.2", git = "https://github.com/antonilol/rust-bitcoincore-zmq.git", features = ["async"] }
 env_logger = "0.11"
+fastrand = "2.0.1"
```

Comment:
This doesn't appear to be used?

Also generally prefer standard libraries over new dependencies.